frontend/src/pages/MainPage.tsx

```diff
@@ -22,12 +24,36 @@ function MainPage() {
   const [celebId, setCelebId] = useState<Celeb['id']>(-1);
   const [restaurantCategory, setRestaurantCategory] = useState<RestaurantCategory>('전체');
   const [hoveredId, setHoveredId] = useState<number | null>(null);
+  const [celebOptions, setCelebOptions] = useState<Celeb[]>();
 
-  const { data, isLoading, refetch } = useQuery<RestaurantListData>({
+  const {
+    data: restaurantDataList,
```

Comment:
바꿔서 써줄거면 타입과 이름을 맞춰주는것이 좋을 것 같아요! 실제로 해당 데이터는 List 가 아니니까요!

```suggestion
    data: restaurantListData,
```